assets/models/system/distilbert-base-cased/spec.yaml

```diff
@@ -6,83 +6,112 @@ properties:
-      Standard_E48s_v3,
-      Standard_E64s_v3,
-      Standard_NC4as_T4_v3,
-      Standard_NC6s_v3,
-      Standard_NC8as_T4_v3,
-      Standard_NC12s_v3,
-      Standard_NC16as_T4_v3,
-      Standard_NC24s_v3,
-      Standard_NC64as_T4_v3,
-      Standard_NC24ads_A100_v4,
-      Standard_NC48ads_A100_v4,
-      Standard_NC96ads_A100_v4,
-      Standard_ND96asr_v4,
-      Standard_ND96amsr_A100_v4,
-      Standard_ND40rs_v2
-    ]
-  finetune_compute_allow_list:
-    [
-      Standard_NC24s_v3,
-      Standard_NC24rs_v3,
-      Standard_ND40rs_v2,
-      Standard_ND96amsr_A100_v4,
-      Standard_ND96asr_v4,
+      Standard_E4s_v3
+      , Standard_E8s_v3
+      , Standard_E16s_v3
+      , Standard_E16s_v3
+      , Standard_E32s_v3
+      , Standard_E32s_v3
+      , Standard_E48s_v3
+      , Standard_E48s_v3
+      , Standard_E64s_v3
+      , Standard_E64s_v3
+      , Standard_FX4mds
+      , Standard_FX4mds
+      , Standard_FX12mds
+      , Standard_FX12mds
+      , Standard_F16s_v2
+      , Standard_F16s_v2
+      , Standard_F32s_v2
+      , Standard_F32s_v2
+      , Standard_F48s_v2
+      , Standard_F48s_v2
+      , Standard_F64s_v2
+      , Standard_F64s_v2
+      , Standard_F72s_v2
+      , Standard_F72s_v2
+      , Standard_FX24mds
+      , Standard_FX24mds
+      , Standard_FX36mds
+      , Standard_FX36mds
+      , Standard_FX48mds
+      , Standard_FX48mds
+      , Standard_DS5_v2
+      , Standard_DS5_v2
+      , Standard_DS12_v2
+      , Standard_DS12_v2
+      , Standard_D8a_v4
+      , Standard_D8a_v4
+      , Standard_D8as_v4
+      , Standard_D8as_v4
+      , Standard_D16a_v4
+      , Standard_D16a_v4
+      , Standard_D16as_v4
+      , Standard_D16as_v4
+      , Standard_D32a_v4
+      , Standard_D32a_v4
+      , Standard_D32as_v4
+      , Standard_D32as_v4
+      , Standard_D48a_v4
+      , Standard_D48a_v4
+      , Standard_D48as_v4
+      , Standard_D48as_v4
+      , Standard_D64a_v4
+      , Standard_D64a_v4
+      , Standard_D64as_v4
+      , Standard_D64as_v4
+      , Standard_D96a_v4
+      , Standard_D96a_v4
+      , Standard_D96as_v4
+      , Standard_D96as_v4
+      , Standard_NC4as_T4_v3
+      , Standard_NC4as_T4_v3
+      , Standard_NC6s_v3
+      , Standard_NC6s_v3
+      , Standard_NC8as_T4_v3
+      , Standard_NC8as_T4_v3
+      , Standard_NC12s_v3
+      , Standard_NC12s_v3
+      , Standard_NC16as_T4_v3
+      , Standard_NC16as_T4_v3
+      , Standard_NC24s_v3
+      , Standard_NC24s_v3
+      , Standard_NC24ads_A100_v4
+      , Standard_NC24ads_A100_v4
+      , Standard_NC48ads_A100_v4
+      , Standard_NC48ads_A100_v4
+      , Standard_NC64as_T4_v3
+      , Standard_NC64as_T4_v3
+      , Standard_NC96ads_A100_v4
+      , Standard_NC96ads_A100_v4
+      , Standard_ND40rs_v2
+      , Standard_ND40rs_v2
+      , Standard_ND96asr_v4
+      , Standard_ND96asr_v4
+      , Standard_ND96amsr_A100_v4
+      , Standard_ND96amsr_A100_v4
     ]
   inference_compute_allow_list:
     [
+      Standard_E2s_v3,
```

Comment:
Please make sure the compute allow lists and the recommended skus have the same values

Reply:
considered the list from prod model card and kept same for both tags and recommendations.